src/index.html

```diff
@@ -2,13 +2,59 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="header__logo logo">
+          <img 
+            src="images/logo.png" 
+            alt="Logo" 
+            class="logo__img"
+          >
+        </a>
+        <nav class="header__menu menu">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a class="menu__link is-active" href="#">Apple</a> 
```

Comment:
```suggestion
              <a class="menu__link is-active" href="#apple">Apple</a> 
```


you can add appropriate href